usagov/usagov-2021
# USAGOV-1087-Social-Media-icon-overlap

## Jira Task

https://cm-jira.usa.gov/browse/USAGOV-1087

## How to Test
 - Go to the footer on any page
 - Open dev tools
 - Delete the <span id="footer-phone"> element
 - Confirm that the social media icons do not overlap with their heading

## Checklist for the Developer



## Checklist for the Peer Reviewers
